src/EditorFeatures/Core/Implementation/BraceMatching/ClassificationTypeDefinitions.cs

```diff
@@ -6,6 +6,8 @@
 using Microsoft.VisualStudio.Text.Classification;
 using Microsoft.VisualStudio.Utilities;
 
+#pragma warning disable 649
```

Comment:
@KevinH-MS it can and does for the full build.  But WPF does an intermediate build doesn't process our `<InternalsVisibleTo>` elements in the project file.